### DISCUSSION TITLE
How to upgrade from an existing ISO

### DISCUSSION THREAD

**Discussion by prodyz** [AveYo/MediaCreationTool.bat#76] [Q&A] [answered]
Maybe I didn't understand the Readme well or my English is to bad. I have an existing ISO. Where have I to place it? And how can I upgrade W10 with it? I placed my ISO in several folders, but it wasn't detected.
TIA Prodyz

**Comment by AveYo** [comment]
you must double-click the iso, then it will be mounted as a new drive letter

https://www.windowscentral.com/how-mount-or-unmount-iso-images-windows-10

**Comment by prodyz** [comment, marked as answer]
The Readme for a dummy like me is not explicit enough. By trial and error I now found the solution myself..
Therefore I would like to suggest a change of the Readme, section "Already have a 11 ISO....":
1. step - Start the Quick_11_iso_esd_wim_TPM_toggle.bat in folder "bypass11". You get the message " No input ........to be patched. Use 'Send to' context menu". Now there will be the new entry "Quick_11_iso_esd_wim_TPM_toggle.bat" in the "Send to" menu item
2.  step - you close the window, go to your ISO-file, right click on it and choose  "Send to", choose "Quick_11_iso_esd_wim_TPM_toggle.bat". Now your 11 ISO will be patched.
3. Mount your 11 ISO and start the installation.

**Comment by AveYo** [comment]
that's not the question you've asked, you did not mention any specific script and I don't have a crystal ball  
just like https://github.com/AveYo/MediaCreationTool.bat/issues/71

> I don't understand your request either. Do you already have that esd file? Are you asking how to use `Quick_11_iso_esd_wim_TPM_toggle.bat` on it? If that's the case:
> 
>     * run the script once, it will add an entry to the `Send to` menu
> 
>     * go to the esd file you have via file explorer, right-click it, Send to, `Quick_11_iso_esd_wim_TPM_toggle.bat`
> 
>     * script will show the image xml on screen and `:D TPM patch added`
> 
>     * if you repeat the steps, script will undo the changes

**Reply by AveYo** [reply]
but that's mostly for use on another pc / clean install
the other bypass script works globally for any type of upgrade
